web/app/themes/ppj/page-footer.php

```diff
@@ -43,7 +43,9 @@
                 <div class="footer">
                     <div class="footer__container">
                         <?= $footerContent; ?>
-                        <div class="footer__logo"></div>
+                        <div class="footer__logo">
+                            <img src="/app/themes/ppj/dest/img/svg/logo.svg" alt="HM Prison and Probation Services logo"/>
```

Comment:
It'd be preferable to use WordPress' [`get_template_directory_uri`](https://developer.wordpress.org/reference/functions/get_template_directory_uri/) function here, rather than hardcoding the URL path to the theme.

```html
<img src="<?= get_template_directory_uri() ?>/dest/img/svg/logo.svg">
```

Reply:
done